core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -1272,6 +1274,7 @@ class ReplicaManagerTest {
       }
 
       verify(mockLogMgr).finishedInitializingLog(ArgumentMatchers.eq(tp), any())
+      assertTrue(mockBrokerEpochSupplier.getCounter() > 0)
```

Comment:
This one does not bring much value either. I would remove it.

Reply:
Done.